batcher/aligned-batcher/src/lib.rs

```diff
@@ -856,7 +856,7 @@ async fn send_message<T: Serialize>(
     ws_conn_sink: Arc<RwLock<SplitSink<WebSocketStream<TcpStream>, Message>>>,
     message: T,
 ) {
-    let serialized_response = serde_json::to_vec(&message).expect("Could not serialize response");
+    let serialized_response = cbor_serialize(&message).expect("Could not serialize response");
```

Comment:
and this one

Reply:
Fixed [f0f5354](https://github.com/yetanotherco/aligned_layer/pull/746/commits/f0f53540a95ec50c447ed30c61ff3eefb02c3701)